src/ir/op.cc

```diff
@@ -122,6 +124,19 @@ TVM_REGISTER_GLOBAL("ir.RegisterOpAttr")
       }
     });
 
+TVM_REGISTER_GLOBAL("ir.RegisterOpLowerIntrinsic")
```

Comment:
I assume this should be put on `src/tir/op.cc`?

Reply:
Yeah I have the same concern but I checked both files and found most similar functions like `RegisterOpAttr` function in the same file. Therefore, I think it better to keep the function here or we would need to migrate all related functions.